### PR TITLE
Update emni.txt

### PR DESCRIPTION
**Pull request by sdec18**
tui kotha theke english poracho babu?
emni kore lekh bhalo lagbe

### CODE REVIEW

File: emni.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Hey anyone 
-will this be like this or needs a change?
+Hey someone
```
Comment (Codesmaster-lab) [inline emni.txt:1]: Ekhane someone noi sobai dekbe tai anyone ae hve

File: emni.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Hey anyone 
-will this be like this or needs a change?
+Hey someone 
+Will this be like this or needs a change?
 thanks and regards 
-Codesmadster
+Codesmaster
```
Comment (Codesmaster-lab) [inline emni.txt:4]: thanks ami pagol hole tui bachiye nis lol :0

Comment (Codesmaster-lab) [inline emni.txt:4]: ami pagol noi thanku @sdec18

**Review by Codesmaster-lab — CHANGES_REQUESTED** [review]

File: emni.txt (hunk 2 above)
Comment (sdec18) [reply emni.txt:4]: ok thanks and regards

File: emni.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (sdec18) [reply emni.txt:1]: acha korchi

> ewfnewljnfnwlfnw
`#include <stdio.h>
`

**Review by Codesmaster-lab — APPROVED** [review]
ok well done bois played well

**Pull request merged**